api_pagopa.yaml

```diff
@@ -0,0 +1,264 @@
+      denominazioneBeneficiario:
+        type: string
+        minLength: 1
+        maxLength: 70
+      codiceUnitOperBeneficiario:
+        type: string
+        minLength: 1
+        maxLength: 35
+      denomUnitOperBeneficiario:
+        type: string
+        minLength: 1
+        maxLength: 70
+      indirizzoBeneficiario:
+        type: string
+        minLength: 1
+        maxLength: 70
+      civicoBeneficiario:
+        type: string
+        minLength: 1
+        maxLength: 16
+      capBeneficiario:
+        type: string
+        minLength: 1
+        maxLength: 16
+      localitaBeneficiario:
+        type: string
+        minLength: 1
+        maxLength: 35
+      provinciaBeneficiario:
+        type: string
+        minLength: 1
+        maxLength: 35
+      nazioneBeneficiario:
+        type: string
+        pattern: '[A-Z]{2}'
+    required:
+      - identificativoUnivocoBeneficiario
+      - denominazioneBeneficiario
+      - codiceUnitOperBeneficiario
+      - denomUnitOperBeneficiario
+      - indirizzoBeneficiario
+      - civicoBeneficiario
+      - capBeneficiario
+      - localitaBeneficiario
+      - provinciaBeneficiario
+      - nazioneBeneficiario
+  Iban:
+    type: string
+    pattern: '[a-zA-Z]{2}[0-9]{2}[a-zA-Z0-9]{1,30}'
+  ImportoEuroCents:
+    description: Amount for payments, in eurocents
+    type: integer
+    minimum: 0
+    maximum: 99999999
+  PaymentActivationsGetResponse:
+    type: object
+    description: Define the response to send to App to provide the payment activation status related to a codiceContestoPagamento
+    properties: 
+      idPagamento:
+        type: string
+        minLength: 1
+        maxLength: 35
+    required:
+    - idPagamento
+  PaymentActivationsPostRequest:
+    type: object
+    description: Define the request received from CD App to require a payment activation
+    properties:
+      rptId:
+        $ref: '#/definitions/RptIdFromString'
+      importoSingoloVersamento:
+        $ref: '#/definitions/ImportoEuroCents'
+      codiceContestoPagamento:
+        $ref: '#/definitions/CodiceContestoPagamento'
+    required:
+      - rptId
+      - importoSingoloVersamento
+      - codiceContestoPagamento
+  PaymentActivationsPostResponse:
+    type: object
+    description: Define the response to send to CD App containing payment information
+    properties:      
+      importoSingoloVersamento:
+        $ref: '#/definitions/ImportoEuroCents'
+      ibanAccredito:
+        $ref: '#/definitions/Iban'
+      causaleVersamento:
+        type: string
+        minLength: 1
+        maxLength: 140
+      enteBeneficiario:
+        $ref: '#/definitions/EnteBeneficiario'
+      spezzoniCausaleVersamento:
+        $ref: '#/definitions/SpezzoniCausaleVersamento'
+    required:
+    - importoSingoloVersamento
+  PaymentRequestsGetResponse:
+    type: object
+    title: PaymentRequestsGetResponse
+    description: Define the response to send to CD App containing payment information
```

Comment:
i looks like it's the same description as `PaymentActivationsPostResponse` above. How these ones differ ?